src/Page/Definition.elm

```diff
@@ -332,12 +333,15 @@ expanderDefinitionStyles =
     ]
 
 
-quoteStyles : List Style
-quoteStyles =
-    [ batch
-        [ fontSize (rem 1.1)
+quoteStyle : Style
+quoteStyle =
+    batch
+        [ borderLeft3 (px 5) solid colours.midgrey
+        , borderRadius (px 5)
+        , fontSize (rem 1.1)
+        , fontStyle italic
         , fontWeight (int 300)
+        , paddingLeft (px 10)
+        , before [ property "content" "'\"'" ]
```

Comment:
I think grammatically should use quotes or italics - but not both. I am happy with italics as it livens up the visuals a little.